Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -89,3 +91,54 @@
     line height remains unchanged. */
   content: "\200b";
 }
+
+.bocEnumValue input[type=radio]
```

Comment:
Please check, I think we can move this whole logic to the Common.css and apply to to all input[type=checkbox|radiobutton] that have remotion-themed applied or are nested in remotion-themed.

We expect/require that these elements are followed by a label/span.

Please don't forget to move the icons to the Remotion.Web sprite.

In the BocList, you will also find occurances of checkbox/radiobutton. Those should be automatically covered once we use the remotion-themed approach.

Reply:
```css
.remotion-themed input[type="radio"],
.remotion-themed input[type="checkbox"],
input[type="radio"] .remotion-themed,
input[type="checkbox"] .remotion-themed
```

If I put this in the `Common.css` the default `checkbox` and `radiobutton` in the `boclist` would also be affected.

Reply:
```
input[type="radio"] .remotion-themed,
input[type="checkbox"] .remotion-themed
```
Those two need to go, then there is no conflict. Same as with input[type=text], select, etc

Reply:
No, in both cases the `remotion-themed` class comes before the `input` element, so I think there is no way of doing this without adding additional selectors. Maybe this should not go inside the `Common.css`.

Reply:
I'm sorry, but then I don't understand the problem because if the input is inside remotion-themed, we want to have the theme. Please elaborate where this is not the intended outcome.

Reply:
Alright, the problem is that the `bocList` cell contains a default `checkbox` and no `label`, therefore when adding the rule inside the `Common.css` the `checkbox` just disappears.

![image](https://user-images.githubusercontent.com/48289242/134476329-06be15e2-39a2-45ed-8feb-871da9a67d98.png)

![image](https://user-images.githubusercontent.com/48289242/134476594-cb318f4b-bc1c-4694-865d-764d0ed2d242.png)

Reply:
correct. this needs to be updated to add an empty span that covers this part.